pyvo/registry/regtap.py

```diff
@@ -839,12 +874,52 @@ def service(self):
         constrained the service type; otherwise, many resources will
         have multiple capabilities.  Use get_service instead in
         such cases.
+
+        See Also
+        --------
+        get_service : select a service of a specific service type.
+        list_services : return a list with all the available services.
         """
         if self._service is not None:
             return self._service
         self._service = self.get_service(None, True)
         return self._service
 
+    def list_services(self, service_type=None):
```

Comment:
This is borderline, `service_type` is optional, so strictly speaking we could force the users to spell it out as keyword, but also it's obviously just one argument, so I leave it to your judgement to add a `*` or not

Reply:
The only reason I would see to make this keyword only is if we'd expect more arguments to be added later. Right now I cannot imagine which one so I'd leave that as a 'normal' argument. But I don't have a strong opinion for or against.

Note for later when I'll rebase after #507: add type hint here to keep `regtap.py` consistent